doc/display.xml

```diff
@@ -170,3 +170,79 @@ gap> FileString("dot/star.dot", DotSymmetricDigraph(gr));
     </Description>
   </ManSection>
 <#/GAPDoc>
+
+<#GAPDoc Label="DotPartialOrderDigraph">
+  <ManSection>
+    <Attr Name="DotPartialOrderDigraph" Arg="digraph"/>
+    <Returns>A string.</Returns>
+    <Description>
+      This function produces a graphical representation of a partial order
+      digraph <A>digraph</A>. <C>DotPartialOrderDigraph</C> will return an error
+      if <A>digraph</A> is not a partial order digraph. See <Ref
+        Prop="IsPartialOrderDigraph"/>.<P/>
+
+      Vertices are displayed as circles, numbered consistently with
+      <A>digraph</A>. Since <A>digraph</A> is a partial order, it is both
```

Comment:
Are you happy with
Since <A>digraph</A> is a partial order, it is both reflexive and transitive. The output of `<C>DotPartialOrderDigraph</C>` is the `<Ref Attr="DotDigraph"/>` of the `<Ref Oper="DigraphReflexiveTransitiveReduction"/>` of `<A>digraph</A>`.

Reply:
Yes